set.go

```diff
@@ -253,3 +253,11 @@ func NewThreadUnsafeSetFromMapKeys[T comparable, V any](val map[T]V) Set[T] {
 
 	return s
 }
+
+func Values[T comparable](s Set[T]) func(func(element T) bool) {
```

Comment:
Since this is on the interface, would you mind adding the respective comments for the `Values` method?

Reply:
Comment is added in 24f21dd8caecdd4ecea9a21e6fa5a437ef369841